ConsoleApp/Car.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+    class Car
+    {
+        public string Brand { get; set; }
+        public string Age { get; set; }
+
+        protected virtual void ignition()
```

Comment:
method name should be
```suggestion
        protected virtual void Ignition()
```